common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -95,6 +99,10 @@
             GenericAccount,
             GenericRefreshToken> mAccountCredentialAdapter;
 
+    @Getter
+    @Accessors(prefix = "m")
+    private final Context mContext;
```

Comment:
This seems like it's common to all of these classes.  Should we interpose an AbstractAndroidOAuth2TokenCache here that just holds the context?

Reply:
Let's keep it here for now to keep the change minimum.(They'll be 'abstracted out' anyway).